source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -74,6 +74,7 @@ class FilterConfig {
         pool_(scope_.symbolTable()),
         metadata_context_namespaces_(config.metadata_context_namespaces().begin(),
                                      config.metadata_context_namespaces().end()),
+        peer_labels_(config.peer_labels().begin(), config.peer_labels().end()),
```

Comment:
I'm fine with this as is but @htuch any thoughts on the API front on if we want to validate for known values
pros: reduces typos
cons: harder to extend for folks who might subclass ext_authz and add their own custom labels